globals.py

```diff
@@ -1,2 +1,16 @@
+import threading
+
+buy_ready = threading.Event()
+sell_ready = threading.Event()
 stop_threads = False
 old_coins = {}
+
+#TRADE_OPTIONS config values
```

Comment:
Is there a particular reason behind moving the config options from yaml file to a python one?